### PR TITLE
Change danish (da) date format to match the rest of the locales

### PR DESCRIPTION
**Pull request by jstoone**
**What is wrong**
We had a bug where everything but the danish date format was localized.

**What solved it**
After manually overwriting the format to be `dd/mm/yyyy` form within the project, it displayed the correct danish format.

**What has changed**
I've only changed the date format from `dd-mm-yyyy` to `dd/mm/yyyy`. Feel free to change the table below if disagree with this not being BC.

| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Related tickets | N/A
| License         | MIT

### CODE REVIEW

File: js/locales/bootstrap-datepicker.da.js [hunk 1]
```diff
@@ -13,6 +13,6 @@
 		today: "I Dag",
 		weekStart: 1,
 		clear: "Nulstil",
-		format: "dd-mm-yyyy"
+        format: "dd/mm/yyyy"
```
Comment (jstoone) [inline js/locales/bootstrap-datepicker.da.js]: Sure, I'll push a new commit soon. :)

**Comment by acrobat** [comment]
@jstoone Can you fix the comment from @Azaret and rebase the branch against the current master? Thanks!

File: js/locales/bootstrap-datepicker.da.js (hunk 1 above)
Comment (acrobat) [inline js/locales/bootstrap-datepicker.da.js]: I will push these changes in a separate commit, so we can merge this!

**Comment by acrobat** [comment]
Thanks @jstoone

**Pull request merged**